lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -1827,10 +1929,16 @@ bool EarlyFunctionAnalysis::runOnModule(Module &M) {
                                  nullptr);
   FunctionAnalysisResults Properties(std::move(DefaultSummary));
 
+  // Cache pre-population
+  Properties.initializeCache(M, ABIRegisters, Binary);
+
   // Instantiate a CFEPAnalyzer object
   using CFEPA = CFEPAnalyzer<FunctionAnalysisResults>;
   CFEPA Analyzer(M, &GCBI, ABIRegisters, Properties, Binary);
 
+  // Re-create fake functions, should they exist
+  Analyzer.initializeFakeFunctions();
```

Comment:
Why let the user call this? Make the function `private` and call it in the constructor.